wrappers/services_test.go

```diff
@@ -470,3 +470,71 @@ func (s *servicesTestSuite) TestStartSnapMultiServicesFailStartCleanup(c *C) {
 		{"show", "--property=ActiveState", svc1Name},
 	})
 }
+
+func (s *servicesTestSuite) TestServiceAfterBefore(c *C) {
+	var sysdLog [][]string
+
+	r := systemd.MockSystemctl(func(cmd ...string) ([]byte, error) {
+		sysdLog = append(sysdLog, cmd)
+		return nil, nil
+	})
+	defer r()
+
+	info := snaptest.MockSnap(c, packageHello+`
+ svc2:
+   daemon: forking
+   after: [svc1]
+ svc3:
+   daemon: forking
+   before: [svc4]
+   after:  [svc2]
+ svc4:
+   daemon: forking
+   after:
+     - svc1
+     - svc2
+     - svc3
+`, contentsHello, &snap.SideInfo{Revision: snap.R(12)})
```

Comment:
Can we please have the string and the method call separated out so that we don't have that awkward ending?
The parameters are way too far from the call otherwise.